pkgs/development/compilers/compcert/default.nix

```diff
@@ -16,15 +16,14 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url    = "http://compcert.inria.fr/release/${name}.tgz";
```

Comment:
This should be more stable https://github.com/AbsInt/CompCert

Hopefully the archives don't defer too much.

Reply:
I believe that both sources are tightly coupled. How could one be more stable than the other?

Reply:
@worldofpeace is saying that by using the Git repository tags to fetch rather than the inria.fr server, there will probably be no force pushes to master which would retroactively invalidate hashes. Which I'd agree with, and probably would make it more stable in the future if this repeatedly happens -- but in this case we don't even know what changed, unfortunately (there are no commits to CompCert since the 3.5 tag was posted, which is odd...)